snforge_std/src/file_operations.cairo

```diff
@@ -37,13 +37,34 @@ fn parse_txt(file: @File) -> Array<felt252> {
     result
 }
 
-trait TxtParser<T, impl TSerde: Serde<T>> {
+fn read_json(file: @File) -> Array<felt252> {
+    let content = cheatcode::<'read_json'>(array![*file.path].span());
+
+    let mut result = array![];
+
+    let mut i = 0;
+    loop {
+        if content.len() == i {
+            break ();
+        }
+        result.append(*content[i]);
+        i += 1;
+    };
+    result
+}
+
+trait Parser<T, impl TSerde: Serde<T>> {
     fn deserialize_txt(file: @File) -> Option<T>;
+    fn deserialize_json(file: @File) -> Option<T>;
 }
 
-impl TxtParserImpl<T, impl TSerde: Serde<T>> of TxtParser<T> {
+impl ParserImpl<T, impl TSerde: Serde<T>> of Parser<T> {
     fn deserialize_txt(file: @File) -> Option<T> {
-        let mut content = cheatcode::<'parse_txt'>(array![*file.path].span());
+        let mut content = cheatcode::<'read_txt'>(array![*file.path].span());
+        Serde::<T>::deserialize(ref content)
+    }
+    fn deserialize_json(file: @File) -> Option<T> {
```

Comment:
The name is not really fitting here, it loads + deserializes (deserialization seems a bit extra since it's only a call to serde, but i can live with that)

Reply:
Previously we had `parse_txt` (so it'd be `parse_json` here) which is also not ideal, because we are also loading.

Perhaps there's some other option than parse/load/deserialize, though I don't have any idea atm.